packages/fast-components/src/button/anchor.ts

```diff
@@ -0,0 +1,112 @@
+import { attr, FastElement, html, ref } from "@microsoft/fast-element";
+import { ButtonAppearance } from "./button";
+
+export type AnchorAppearance = ButtonAppearance | "hypertext";
+
+export const anchorTemplate = html<Anchor>`
+    <a
+        class="control"
+        download="${x => x.download}"
+        href="${x => x.href}"
+        hreflang="${x => x.hreflang}"
+        ping="${x => x.lang}"
+        referrerpolicy="${x => x.referrerpolicy}"
+        rel="${x => x.rel}"
+        target="${x => x.target}"
+        type="${x => x.type}"
+    >
+        <span
+            name="start"
+            part="start"
+            ${ref("startContainer")}
+        >
+            <slot
+                name="start"
+                ${ref("start")}
+                @slotchange=${x => x.handleStartContentChange()}
+            ></slot>
+        </span>
+        <span class="content" part="content">
+            <slot></slot>
+        </span>
+        <span
+            name="end"
+            part="end"
+            ${ref("endContainer")}
+        >
+            <slot
+                name="end"
+                ${ref("end")}
+                @slotchange=${x => x.handleEndContentChange()}
+            ></slot>
+        </span>
+    </a>
+`;
+
+export class Anchor extends FastElement {
+    @attr
+    public appearance: AnchorAppearance;
+    public appearanceChanged(): void {
```

Comment:
this method gets prev and next values: `public appearanceChanged(prev: string, next: string)` so I don't think you need to track a private currentAppearance

Reply:
Sweet, I’ll update to use the API.